src/cdk/platform/platform.ts

```diff
@@ -37,11 +39,11 @@ export class Platform {
   // Webkit is part of the userAgent in EdgeHTML, Blink and Trident. Therefore we need to
   // ensure that Webkit runs standalone and is not used as another engine's base.
   WEBKIT: boolean = this.isBrowser &&
-      /AppleWebKit/i.test(navigator.userAgent) && !this.BLINK && !this.EDGE && !this.TRIDENT;
+    /AppleWebKit/i.test(navigator.userAgent) && !this.BLINK && !this.EDGE && !this.TRIDENT;
```

Comment:
The indent was right before (+4 for overflow)

Reply:
Will fix here and other places